### ISSUE TITLE
Admin v1.10 RC7 - Failure to fetch plugins or themes

### ISSUE THREAD

**Issue by Verequies** [getgrav/grav-plugin-admin#1851]
After installing the latest Grav + Admin RC, I am unable to install plugins or themes. Installing from the terminal works perfectly fine, however when attempting to do so from the admin panel I get:
```Fetch Failed: {"status":"success","dependencies":[]}```
Changing the fetch method from 'Auto' to 'fopen' or cURL' does not help. 
Not sure if it helps, but the PHP version installed is PHP 5.4.

Looking at the console, it shows 403 accessing ```https://getgrav.org/images/```.

**Comment by mahagr** [comment]
Check the PHP version again, Grav minimum requirement is PHP 7.1.

**Comment by Verequies** [comment]
Definitely is PHP v7.4.3. No doubt - running Arch Linux.

**Comment by rhukster** [comment]
can you visit the `configuration -> info` panel and paste the output in a code block?

**Comment by Verequies** [comment]
How much info do you need? Don't particularly want to expose my entire configuration.

**Comment by rhukster** [comment]
the php build setup really, fopen, curl stuff, things that could impact it. Most of it is pretty safe, just remove any IPs or paths if you are concerned

**Comment by Verequies** [comment]
Alright, well this is the entire page minus any paths and stuff. Perhaps an export option would be handy to keep formatting?

```
PHP Version 7.4.3
 
System
Linux Example 5.4.24.a-1-hardened #1 SMP PREEMPT Thu, 05 Mar 2020 19:28:01 +0000 x86_64
Build Date
Feb 18 2020 15:35:13
Configure Command
'./configure' '--srcdir=../php-7.4.3' '--config-cache' '--prefix=/usr' '--sbindir=/usr/bin' '--sysconfdir=/etc/php' '--localstatedir=/var' '--with-layout=GNU' '--with-config-file-path=/etc/php' '--with-config-file-scan-dir=/etc/php/conf.d' '--disable-rpath' '--mandir=/usr/share/man' '--enable-cgi' '--enable-fpm' '--with-fpm-systemd' '--with-fpm-acl' '--with-fpm-user=http' '--with-fpm-group=http' '--enable-embed=shared' '--enable-bcmath=shared' '--enable-calendar=shared' '--enable-dba=shared' '--enable-exif=shared' '--enable-ftp=shared' '--enable-gd=shared' '--enable-intl=shared' '--enable-mbstring' '--enable-pcntl' '--enable-shmop=shared' '--enable-soap=shared' '--enable-sockets=shared' '--enable-sysvmsg=shared' '--enable-sysvsem=shared' '--enable-sysvshm=shared' '--with-bz2=shared' '--with-curl=shared' '--with-db4=/usr' '--with-enchant=shared' '--with-external-gd' '--with-external-pcre' '--with-ffi=shared' '--with-gdbm' '--with-gettext=shared' '--with-gmp=shared' '--with-iconv=shared' '--with-imap-ssl' '--with-imap=shared' '--with-kerberos' '--with-ldap=shared' '--with-ldap-sasl' '--with-mhash' '--with-mysql-sock=/run/mysqld/mysqld.sock' '--with-mysqli=shared,mysqlnd' '--with-openssl' '--with-password-argon2' '--with-pdo-dblib=shared,/usr' '--with-pdo-mysql=shared,mysqlnd' '--with-pdo-odbc=shared,unixODBC,/usr' '--with-pdo-pgsql=shared' '--with-pdo-sqlite=shared' '--with-pgsql=shared' '--with-pspell=shared' '--with-readline' '--with-snmp=shared' '--with-sodium=shared' '--with-sqlite3=shared' '--with-tidy=shared' '--with-unixODBC=shared' '--with-xmlrpc=shared' '--with-xsl=shared' '--with-zip=shared' '--with-zlib'
Server API
FPM/FastCGI
Virtual Directory Support
disabled
Configuration File (php.ini) Path
/etc/php
Loaded Configuration File
/etc/php/php.ini
Scan this dir for additional .ini files
/etc/php/conf.d
Additional .ini files parsed
/etc/php/conf.d/apcu.ini, /etc/php/conf.d/yaml.ini
PHP API
20190902
PHP Extension
20190902
Zend Extension
320190902
Zend Extension Build
API320190902,NTS
PHP Extension Build
API20190902,NTS
Debug Build
no
Thread Safety
disabled
Zend Signal Handling
enabled
Zend Memory Manager
enabled
Zend Multibyte Support
provided by mbstring
IPv6 Support
enabled
DTrace Support
disabled
Registered PHP Streams
https, ftps, compress.zlib, php, file, glob, data, http, ftp, compress.bzip2, zip, phar, image, page, account, system, user, environment, asset, blueprints, config, plugins, plugin, themes, languages, cache, log, backup, tmp, user-data, theme
Registered Stream Socket Transports
tcp, udp, unix, udg, ssl, tls, tlsv1.0, tlsv1.1, tlsv1.2, tlsv1.3
Registered Stream Filters
zlib.*, string.rot13, string.toupper, string.tolower, string.strip_tags, convert.*, consumed, dechunk, bzip2.*
 
This program makes use of the Zend Scripting Language Engine:
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.3, Copyright (c), by Zend Technologies
Configuration
apc
APC Compatibility
1.0.5
apcu
APCu Support
Enabled
Version
5.1.18
APCu Debugging
Disabled
MMAP Support
Enabled
MMAP File Mask
no value
Serialization Support
php
Build Date
Nov 29 2019 17:36:52
 
Directive
Local Value
Master Value
apc.coredump_unmap
Off
Off
apc.enable_cli
Off
Off
apc.enabled
On
On
apc.entries_hint
4096
4096
apc.gc_ttl
3600
3600
apc.mmap_file_mask
no value
no value
apc.preload_path
no value
no value
apc.serializer
php
php
apc.shm_segments
1
1
apc.shm_size
513M
513M
apc.slam_defense
Off
Off
apc.smart
0
0
apc.ttl
0
0
apc.use_request_time
On
On
bz2
BZip2 Support
Enabled
Stream Wrapper support
compress.bzip2://
Stream Filter support
bzip2.decompress, bzip2.compress
BZip2 Version
1.0.8, 13-Jul-2019
cgi-fcgi
php-fpm
active
 
Directive
Local Value
Master Value
cgi.discard_path
0
0
cgi.fix_pathinfo
1
1
cgi.force_redirect
1
1
cgi.nph
0
0
cgi.redirect_status_env
no value
no value
cgi.rfc2616_headers
0
0
fastcgi.error_header
no value
no value
fastcgi.logging
1
1
fpm.config
no value
no value
Core
PHP Version
7.4.3
 
Directive
Local Value
Master Value
allow_url_fopen
On
On
allow_url_include
Off
Off
arg_separator.input
&
&
arg_separator.output
&
&
auto_append_file
no value
no value
auto_globals_jit
On
On
auto_prepend_file
no value
no value
browscap
no value
no value
default_charset
UTF-8
UTF-8
default_mimetype
text/html
text/html
disable_classes
no value
no value
disable_functions
no value
no value
display_errors
Off
Off
display_startup_errors
Off
Off
doc_root
no value
no value
docref_ext
no value
no value
docref_root
no value
no value
enable_dl
Off
Off
enable_post_data_reading
On
On
error_append_string
no value
no value
error_log
no value
no value
error_prepend_string
no value
no value
error_reporting
22527
22527
expose_php
Off
Off
extension_dir
/usr/lib/php/modules/
/usr/lib/php/modules/
file_uploads
On
On
hard_timeout
2
2
highlight.comment
#FF8000
#FF8000
highlight.default
#0000BB
#0000BB
highlight.html
#000000
#000000
highlight.keyword
#007700
#007700
highlight.string
#DD0000
#DD0000
html_errors
On
On
ignore_repeated_errors
Off
Off
ignore_repeated_source
Off
Off
ignore_user_abort
Off
Off
implicit_flush
Off
Off
include_path
.:
.:
input_encoding
no value
no value
internal_encoding
no value
no value
log_errors
On
On
log_errors_max_len
1024
1024
mail.add_x_header
Off
Off
mail.force_extra_parameters
no value
no value
mail.log
no value
no value
max_execution_time
300
300
max_file_uploads
20
20
max_input_nesting_level
64
64
max_input_time
300
300
max_input_vars
1000000
1000000
memory_limit
512M
512M
open_basedir
no value
no value
output_buffering
4096
4096
output_encoding
no value
no value
output_handler
no value
no value
post_max_size
65M
65M
precision
14
14
realpath_cache_size
4096K
4096K
realpath_cache_ttl
120
120
register_argc_argv
Off
Off
report_memleaks
On
On
report_zend_debug
On
On
request_order
GP
GP
sendmail_from
no value
no value
sendmail_path
/usr/sbin/sendmail -t -i -f administrator@example.com
/usr/sbin/sendmail -t -i -f administrator@example.com
serialize_precision
-1
-1
short_open_tag
Off
Off
SMTP
localhost
localhost
smtp_port
25
25
sys_temp_dir
no value
no value
syslog.facility
LOG_USER
LOG_USER
syslog.filter
no-ctrl
no-ctrl
syslog.ident
php
php
track_errors
Off
Off
unserialize_callback_func
no value
no value
upload_max_filesize
65M
65M
upload_tmp_dir
/usr/www/example.com/.tmp
/usr/www/example.com/.tmp
user_dir
no value
no value
user_ini.cache_ttl
300
300
user_ini.filename
.user.ini
.user.ini
variables_order
GPCS
GPCS
xmlrpc_error_number
0
0
xmlrpc_errors
Off
Off
zend.assertions
-1
-1
zend.detect_unicode
On
On
zend.enable_gc
On
On
zend.exception_ignore_args
Off
Off
zend.multibyte
Off
Off
zend.script_encoding
no value
no value
zend.signal_check
Off
Off
ctype
ctype functions
enabled
curl
cURL support
enabled
cURL Information
7.69.0
Age
5
Features
AsynchDNS
Yes
CharConv
No
Debug
No
GSS-Negotiate
No
IDN
Yes
IPv6
Yes
krb4
No
Largefile
Yes
libz
Yes
NTLM
Yes
NTLMWB
Yes
SPNEGO
Yes
SSL
Yes
SSPI
No
TLS-SRP
Yes
HTTP2
Yes
GSSAPI
Yes
KERBEROS5
Yes
UNIX_SOCKETS
Yes
PSL
Yes
HTTPS_PROXY
Yes
MULTI_SSL
No
BROTLI
No
Protocols
dict, file, ftp, ftps, gopher, http, https, imap, imaps, pop3, pop3s, rtsp, scp, sftp, smb, smbs, smtp, smtps, telnet, tftp
Host
x86_64-pc-linux-gnu
SSL Version
OpenSSL/1.1.1d
ZLib Version
1.2.11
libSSH Version
libssh2/1.9.0
 
Directive
Local Value
Master Value
curl.cainfo
no value
no value
date
date/time support
enabled
timelib version
2018.03
"Olson" Timezone Database Version
2019.3
Timezone Database
internal
Default timezone
Australia/Brisbane
 
Directive
Local Value
Master Value
date.default_latitude
31.7667
31.7667
date.default_longitude
35.2333
35.2333
date.sunrise_zenith
90.583333
90.583333
date.sunset_zenith
90.583333
90.583333
date.timezone
Australia/Brisbane
Australia/Brisbane
dom
DOM/XML
enabled
DOM/XML API Version
20031129
libxml Version
2.9.10
HTML Support
enabled
XPath Support
enabled
XPointer Support
enabled
Schema Support
enabled
RelaxNG Support
enabled
fileinfo
fileinfo support
enabled
libmagic
537
filter
Input Validation and Filtering
enabled
 
Directive
Local Value
Master Value
filter.default
unsafe_raw
unsafe_raw
filter.default_flags
no value
no value
gd
GD Support
enabled
GD headers Version
2.2.5
GD library Version
2.2.5
FreeType Support
enabled
FreeType Linkage
with freetype
GIF Read Support
enabled
GIF Create Support
enabled
JPEG Support
enabled
PNG Support
enabled
WBMP Support
enabled
XPM Support
enabled
XBM Support
enabled
WebP Support
enabled
BMP Support
enabled
TGA Read Support
enabled
 
Directive
Local Value
Master Value
gd.jpeg_ignore_warning
1
1
hash
hash support
enabled
Hashing Engines
md2 md4 md5 sha1 sha224 sha256 sha384 sha512/224 sha512/256 sha512 sha3-224 sha3-256 sha3-384 sha3-512 ripemd128 ripemd160 ripemd256 ripemd320 whirlpool tiger128,3 tiger160,3 tiger192,3 tiger128,4 tiger160,4 tiger192,4 snefru snefru256 gost gost-crypto adler32 crc32 crc32b crc32c fnv132 fnv1a32 fnv164 fnv1a64 joaat haval128,3 haval160,3 haval192,3 haval224,3 haval256,3 haval128,4 haval160,4 haval192,4 haval224,4 haval256,4 haval128,5 haval160,5 haval192,5 haval224,5 haval256,5
 
MHASH support
Enabled
MHASH API Version
Emulated Support
json
json support
enabled
libxml
libXML support
active
libXML Compiled Version
2.9.10
libXML Loaded Version
20910
libXML streams
enabled
mbstring
Multibyte Support
enabled
Multibyte string engine
libmbfl
HTTP input encoding translation
disabled
libmbfl version
1.3.2
 
mbstring extension makes use of "streamable kanji code filter and converter", which is distributed under the GNU Lesser General Public License version 2.1.
 
Multibyte (japanese) regex support
enabled
Multibyte regex (oniguruma) version
6.9.4
 
Directive
Local Value
Master Value
mbstring.detect_order
no value
no value
mbstring.encoding_translation
Off
Off
mbstring.func_overload
0
0
mbstring.http_input
no value
no value
mbstring.http_output
no value
no value
mbstring.http_output_conv_mimetypes
^(text/|application/xhtml\+xml)
^(text/|application/xhtml\+xml)
mbstring.internal_encoding
no value
no value
mbstring.language
neutral
neutral
mbstring.regex_retry_limit
1000000
1000000
mbstring.regex_stack_limit
100000
100000
mbstring.strict_detection
Off
Off
mbstring.substitute_character
no value
no value
mysqli
MysqlI Support
enabled
Client API library version
mysqlnd 7.4.3
Active Persistent Links
0
Inactive Persistent Links
0
Active Links
0
 
Directive
Local Value
Master Value
mysqli.allow_local_infile
Off
Off
mysqli.allow_persistent
On
On
mysqli.default_host
no value
no value
mysqli.default_port
3306
3306
mysqli.default_pw
no value
no value
mysqli.default_socket
/run/mysqld/mysqld.sock
/run/mysqld/mysqld.sock
mysqli.default_user
no value
no value
mysqli.max_links
Unlimited
Unlimited
mysqli.max_persistent
Unlimited
Unlimited
mysqli.reconnect
Off
Off
mysqli.rollback_on_cached_plink
Off
Off
mysqlnd
mysqlnd
enabled
Version
mysqlnd 7.4.3
Compression
supported
core SSL
supported
extended SSL
supported
Command buffer size
4096
Read buffer size
32768
Read timeout
86400
Collecting statistics
Yes
Collecting memory statistics
No
Tracing
n/a
Loaded plugins
mysqlnd,debug_trace,auth_plugin_mysql_native_password,auth_plugin_mysql_clear_password,auth_plugin_caching_sha2_password,auth_plugin_sha256_password
API Extensions
mysqli
openssl
OpenSSL support
enabled
OpenSSL Library Version
OpenSSL 1.1.1d 10 Sep 2019
OpenSSL Header Version
OpenSSL 1.1.1d 10 Sep 2019
Openssl default config
/etc/ssl/openssl.cnf
 
Directive
Local Value
Master Value
openssl.cafile
no value
no value
openssl.capath
no value
no value
pcntl
pcntl support
enabled
pcre
PCRE (Perl Compatible Regular Expressions) Support
enabled
PCRE Library Version
10.34 2019-11-21
PCRE Unicode Version
12.1.0
PCRE JIT Support
enabled
PCRE JIT Target
x86 64bit (little endian + unaligned)
 
Directive
Local Value
Master Value
pcre.backtrack_limit
1000000
1000000
pcre.jit
1
1
pcre.recursion_limit
100000
100000
PDO
PDO support
enabled
PDO drivers
no value
Phar
Phar: PHP Archive support
enabled
Phar API version
1.1.1
Phar-based phar archives
enabled
Tar-based phar archives
enabled
ZIP-based phar archives
enabled
gzip compression
enabled
bzip2 compression
enabled
Native OpenSSL support
enabled
 
Phar based on pear/PHP_Archive, original concept by Davey Shafik.
Phar fully realized by Gregory Beaver and Marcus Boerger.
Portions of tar implementation Copyright (c) 2003-2009 Tim Kientzle.
 
Directive
Local Value
Master Value
phar.cache_list
no value
no value
phar.readonly
On
On
phar.require_hash
On
On
posix
POSIX support
enabled
readline
Readline Support
enabled
Readline library
8.0
 
Directive
Local Value
Master Value
cli.pager
no value
no value
cli.prompt
\b \> 
\b \> 
Reflection
Reflection
enabled
session
Session Support
enabled
Registered save handlers
files user
Registered serializer handlers
php_serialize php php_binary
 
Directive
Local Value
Master Value
session.auto_start
Off
Off
session.cache_expire
180
180
session.cache_limiter
nocache
nocache
session.cookie_domain
example.com
no value
session.cookie_httponly
1
no value
session.cookie_lifetime
1800
0
session.cookie_path
/
/
session.cookie_samesite
no value
no value
session.cookie_secure
0
0
session.gc_divisor
1000
1000
session.gc_maxlifetime
1440
1440
session.gc_probability
0
0
session.lazy_write
On
On
session.name
grav-site-f85cec3-admin
PHPSESSID
session.referer_check
no value
no value
session.save_handler
files
files
session.save_path
/usr/www/example.com/.tmp
/usr/www/example.com/.tmp
session.serialize_handler
php
php
session.sid_bits_per_character
5
5
session.sid_length
26
26
session.upload_progress.cleanup
On
On
session.upload_progress.enabled
On
On
session.upload_progress.freq
1%
1%
session.upload_progress.min_freq
1
1
session.upload_progress.name
PHP_SESSION_UPLOAD_PROGRESS
PHP_SESSION_UPLOAD_PROGRESS
session.upload_progress.prefix
upload_progress_
upload_progress_
session.use_cookies
1
1
session.use_only_cookies
1
1
session.use_strict_mode
1
0
session.use_trans_sid
0
0
SimpleXML
SimpleXML support
enabled
Schema support
enabled
SPL
SPL support
enabled
Interfaces
OuterIterator, RecursiveIterator, SeekableIterator, SplObserver, SplSubject
Classes
AppendIterator, ArrayIterator, ArrayObject, BadFunctionCallException, BadMethodCallException, CachingIterator, CallbackFilterIterator, DirectoryIterator, DomainException, EmptyIterator, FilesystemIterator, FilterIterator, GlobIterator, InfiniteIterator, InvalidArgumentException, IteratorIterator, LengthException, LimitIterator, LogicException, MultipleIterator, NoRewindIterator, OutOfBoundsException, OutOfRangeException, OverflowException, ParentIterator, RangeException, RecursiveArrayIterator, RecursiveCachingIterator, RecursiveCallbackFilterIterator, RecursiveDirectoryIterator, RecursiveFilterIterator, RecursiveIteratorIterator, RecursiveRegexIterator, RecursiveTreeIterator, RegexIterator, RuntimeException, SplDoublyLinkedList, SplFileInfo, SplFileObject, SplFixedArray, SplHeap, SplMinHeap, SplMaxHeap, SplObjectStorage, SplPriorityQueue, SplQueue, SplStack, SplTempFileObject, UnderflowException, UnexpectedValueException
standard
Dynamic Library Support
enabled
Path to sendmail
/usr/sbin/sendmail -t -i -f administrator@example.com
 
Directive
Local Value
Master Value
assert.active
1
1
assert.bail
0
0
assert.callback
no value
no value
assert.exception
0
0
assert.quiet_eval
0
0
assert.warning
1
1
auto_detect_line_endings
0
0
default_socket_timeout
60
60
from
no value
no value
session.trans_sid_hosts
no value
no value
session.trans_sid_tags
a=href,area=href,frame=src,form=
a=href,area=href,frame=src,form=
unserialize_max_depth
4096
4096
url_rewriter.hosts
no value
no value
url_rewriter.tags
form=
form=
user_agent
no value
no value
tokenizer
Tokenizer Support
enabled
xml
XML Support
active
XML Namespace Support
active
libxml2 Version
2.9.10
xmlreader
XMLReader
enabled
xmlwriter
XMLWriter
enabled
yaml
LibYAML Support
enabled
Module Version
2.0.4
LibYAML Version
0.2.2
 
Directive
Local Value
Master Value
yaml.decode_binary
0
0
yaml.decode_php
0
0
yaml.decode_timestamp
0
0
yaml.output_canonical
0
0
yaml.output_indent
2
2
yaml.output_width
80
80
Zend OPcache
Opcode Caching
Up and Running
Optimization
Enabled
SHM Cache
Enabled
File Cache
Disabled
Startup
OK
Shared memory model
mmap
Cache hits
93729
Cache misses
1609
Used memory
26819248
Free memory
502718600
Wasted memory
7333064
Interned Strings Used memory
4752992
Interned Strings Free memory
1538016
Cached scripts
952
Cached keys
1672
Max keys
16229
OOM restarts
0
Hash keys restarts
0
Manual restarts
4
 
Directive
Local Value
Master Value
opcache.blacklist_filename
no value
no value
opcache.consistency_checks
0
0
opcache.dups_fix
Off
Off
opcache.enable
On
On
opcache.enable_cli
Off
Off
opcache.enable_file_override
Off
Off
opcache.error_log
no value
no value
opcache.file_cache
no value
no value
opcache.file_cache_consistency_checks
1
1
opcache.file_cache_only
0
0
opcache.file_update_protection
2
2
opcache.force_restart_timeout
180
180
opcache.huge_code_pages
Off
Off
opcache.interned_strings_buffer
8
8
opcache.lockfile_path
/tmp
/tmp
opcache.log_verbosity_level
1
1
opcache.max_accelerated_files
10000
10000
opcache.max_file_size
0
0
opcache.max_wasted_percentage
5
5
opcache.memory_consumption
512
512
opcache.opt_debug_level
0
0
opcache.optimization_level
0x7FFEBFFF
0x7FFEBFFF
opcache.preferred_memory_model
no value
no value
opcache.preload
no value
no value
opcache.preload_user
no value
no value
opcache.protect_memory
0
0
opcache.restrict_api
no value
no value
opcache.revalidate_freq
0
0
opcache.revalidate_path
Off
Off
opcache.save_comments
1
1
opcache.use_cwd
On
On
opcache.validate_permission
Off
Off
opcache.validate_root
Off
Off
opcache.validate_timestamps
On
On
zip
Zip
enabled
Zip version
1.15.6
Libzip headers version
1.5.2
Libzip library version
1.6.1
zlib
ZLib Support
enabled
Stream Wrapper
compress.zlib://
Stream Filter
zlib.inflate, zlib.deflate
Compiled Version
1.2.11
Linked Version
1.2.11
 
Directive
Local Value
Master Value
zlib.output_compression
Off
Off
zlib.output_compression_level
-1
-1
zlib.output_handler
no value
no value
Additional Modules
Module Name
Environment
Variable
Value
PATH
/usr/bin
TEMP
/usr/www/example.com/.tmp
TMPDIR
/usr/www/example.com/.tmp
TMP
/usr/www/example.com/.tmp
HOSTNAME
no value
USER
example
HOME
/usr/example
PHP Variables
Variable
Value
$_COOKIE['grav-site-f85cec3']
lnureb0kq9mf6n4ol81i8qcmei
$_COOKIE['grav-site-f85cec3-admin']
8isgrst2k6h38f483dqd3pbvgu
$_SERVER['PATH']
/usr/bin
$_SERVER['TEMP']
/usr/www/example.com/.tmp
$_SERVER['TMPDIR']
/usr/www/example.com/.tmp
$_SERVER['TMP']
/usr/www/example.com/.tmp
$_SERVER['HOSTNAME']
no value
$_SERVER['USER']
example
$_SERVER['HOME']
/usr/example
$_SERVER['HTTP_COOKIE']
grav-site-f85cec3=lnureb0kq9mf6n4ol81i8qcmei; grav-site-f85cec3-admin=8isgrst2k6h38f483dqd3pbvgu
$_SERVER['HTTP_ACCEPT_LANGUAGE']
en-US,en;q=0.9
$_SERVER['HTTP_ACCEPT_ENCODING']
gzip, deflate, br
$_SERVER['HTTP_REFERER']
https://example.com/admin/config
$_SERVER['HTTP_SEC_FETCH_USER']
?1
$_SERVER['HTTP_SEC_FETCH_MODE']
navigate
$_SERVER['HTTP_SEC_FETCH_SITE']
same-origin
$_SERVER['HTTP_ACCEPT']
text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9
$_SERVER['HTTP_SEC_FETCH_DEST']
document
$_SERVER['HTTP_USER_AGENT']
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.84 Safari/537.36
$_SERVER['HTTP_DNT']
1
$_SERVER['HTTP_UPGRADE_INSECURE_REQUESTS']
1
$_SERVER['HTTP_HOST']
example.com
$_SERVER['SCRIPT_FILENAME']
/usr/www/example.com/www/index.php
$_SERVER['REDIRECT_STATUS']
200
$_SERVER['SERVER_NAME']
example.com
$_SERVER['SERVER_PORT']
443
$_SERVER['SERVER_ADDR']
127.0.1.1
$_SERVER['REMOTE_PORT']
44894
$_SERVER['REMOTE_ADDR']
127.0.0.1
$_SERVER['SERVER_SOFTWARE']
nginx/1.17.8
$_SERVER['GATEWAY_INTERFACE']
CGI/1.1
$_SERVER['HTTPS']
on
$_SERVER['REQUEST_SCHEME']
https
$_SERVER['SERVER_PROTOCOL']
HTTP/2.0
$_SERVER['DOCUMENT_ROOT']
/usr/www/example.com/www
$_SERVER['DOCUMENT_URI']
/index.php
$_SERVER['REQUEST_URI']
/admin/config/info
$_SERVER['SCRIPT_NAME']
/index.php
$_SERVER['CONTENT_LENGTH']
no value
$_SERVER['CONTENT_TYPE']
no value
$_SERVER['REQUEST_METHOD']
GET
$_SERVER['QUERY_STRING']
no value
$_SERVER['FCGI_ROLE']
RESPONDER
$_SERVER['PHP_SELF']
/index.php
$_SERVER['REQUEST_TIME_FLOAT']
1584587345.4934
$_SERVER['REQUEST_TIME']
1584587345
```

**Comment by Verequies** [comment]
Still an issue on RC8

**Comment by mahagr** [comment]
I'm not sure what makes this happen for you, but I'm pretty sure it is something related to your hosting or PHP settings, which can be different between CLI and HTTP.

I'm currently working on the installation, maybe I will find something. That said, I have no idea what might be going wrong there.

**Comment by Verequies** [comment]
Yeah its interesting, since other apps like next cloud work fine in regards to downloading plugins etc.

**Comment by mahagr** [comment]
Is this true also with other Grav servers? Does the older version work?

**Comment by w00fz** [comment]
This is most likely resolved with the current Grav 1.7/ Admin 1.10 - We are now using Symphony's HttpClient wrapper for the downloads and it has proven to be more reliable.

This change was firstly introduced in Grav v1.7.0-rc.10 / Admin v1.10.0-rc.10 and so I would like to know if this has been still an issue for you or not.
